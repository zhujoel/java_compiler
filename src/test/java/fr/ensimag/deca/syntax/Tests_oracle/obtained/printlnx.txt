`> Program
   +> ListDeclClass [List with 0 elements]
   `> Main
      +> ListDeclVar [List with 0 elements]
      `> ListInst [List with 1 elements]
         []> Println
             `> ListExpr [List with 1 elements]
                []> FloatLiteral (5.2256)
