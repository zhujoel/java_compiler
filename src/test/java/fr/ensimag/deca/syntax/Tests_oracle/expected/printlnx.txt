`> Program
   +> ListDeclClass [List with 0 elements]
   `> Main
      +> ListDeclVar [List with 0 elements]
      `> ListInst [List with 1 elements]
         []> Println
             `> ListExpr [List with 1 elements]
                []> Float (5.2256)
